src/client/Microsoft.Identity.Client/MsalException.cs

```diff
@@ -113,6 +114,13 @@ private set
             }
         }
 
+        /// <summary>
+        /// A property bag with extra details for this exception.
+        /// </summary>
+        public IReadOnlyDictionary<string, string> AdditionalResponseParameters { get; set; } 
```

Comment:
`AdditionalErrorData`?
`Response` implies to me that there's an error response from somewhere else (AAD, broker) but if this is intended to be a general property bag, it could be filled by MSAL itself.